runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomPlugin.java

```diff
@@ -95,6 +98,12 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 			return;
 		}
 
+		if ("outerZoomLimit".equals(event.getEventName()) && outerZoomLimit != 128)
```

Comment:
Given that setting intStack values is cheap, is it important for us to have that `outerZoomLimit != 128` check? I'd like to see that check removed, or have `128` defined as a constant somewhere and not just be a magic number.

Reply:
Removed the check